test/Clang/clang_specific_environment.py

```diff
@@ -25,17 +25,25 @@
 __revision__ = "__FILE__ __REVISION__ __DATE__ __DEVELOPER__"
 
 import TestSCons
+from SCons.Environment import Base
 
 _exe = TestSCons._exe
 test = TestSCons.TestSCons()
 
 if not test.where_is('clang'):
     test.skip_test("Could not find 'clang', skipping test.\n")
 
+env_str = "env = Environment(tools=['clang', 'link'])"
+platform = Base()['PLATFORM']
```

Comment:
better to use sys.platform == for this rather than adding additional dependency on Base.. which can change or be removed.

Reply:
agreed, I used the scons way for consistency with other tests, but I would prefer sys.platform.